src/flow-private.h

```diff
@@ -70,6 +70,7 @@ enum {
     FLOW_PROTO_TCP = 0,
     FLOW_PROTO_UDP,
     FLOW_PROTO_ICMP,
+    FLOW_PROTO_ESP,
```

Comment:
the primary use of this would be to have specific timeout values for ESP, which would then have to be set up in `flow.c`. I'm not sure its really needed here, but open to other thoughts.

Reply:
Ah, I see what you mean. Removed for now.